### PR TITLE
Added FAQs - Getting Auth User Profile URL

### PR DESCRIPTION
**Pull request by praisegeek**
Since Auth User profile url isn't included in the response gotten from the /me or /users AD endpoint, it's a common use case to retrieve this details. Using the example in the FAQ, users can get answers to these use cases.

### CODE REVIEW

**Comment by jthake** [comment]
@mtillman is this a standard we use in our concept documents like this? I feel this is useful but not suire the guidance on this.

**Comment by mtillman** [comment]
@jthake I believe @Lauragra may know more than I do. This doesn't appear to be an FAQ as much as it's an example. In principle, it's fine to provide examples as long as it's relevant to the subject and in proximity of what it's an example of.

File: concepts/auth_v2_user.md [hunk 1]
```diff
@@ -219,6 +219,25 @@ A successful token response will look similar to the following.
 | scope |The permissions (scopes) that the access_token is valid for. |
 | refresh_token |A new OAuth 2.0 refresh token. You should replace the old refresh token with this newly acquired refresh token to ensure your refresh tokens remain valid for as long as possible. |
 
+
+## FAQ
```
[the remark this answers is not in the document]
Comment (praisegeek) [reply concepts/auth_v2_user.md]: Ok noted @Lauragra. Updated in 3bc418c

**Comment by Lauragra** [comment]
Reopened a new PR to track this update; closing this one.

**Pull request closed without merging**